src/Jit/Parser.php

```diff
@@ -77,6 +77,7 @@ protected function _parser($content)
         $this->_stream = new TokenStream(['source' => $content, 'wrap' => $this->_states['php']]);
 
         $T_YIELD = defined('HHVM_VERSION') ? 381 : 267;
+        $T_ARROW_FUNCTION = defined('T_FN') ? 343 : 10000;
```

Comment:
The usage of the 10000 integer came from this recommendation:
https://www.php.net/manual/en/tokens.php